angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
+        '@angular-eslint/contextual-decorator': 'error',
+        '@angular-eslint/directive-selector': 'error',
+        '@angular-eslint/no-attribute-decorator': 'error',
+        '@angular-eslint/no-forward-ref': 'error',
+        '@angular-eslint/no-input-prefix': 'error',
+        '@angular-eslint/no-lifecycle-call': 'error',
+        '@angular-eslint/no-pipe-impure': 'error',
+        '@angular-eslint/no-queries-metadata-property': 'error',
+        '@angular-eslint/pipe-prefix': 'error',
+        '@angular-eslint/prefer-on-push-component-change-detection': 'error',
+        '@angular-eslint/prefer-output-readonly': 'error',
```

Comment:
[@angular-eslint/prefer-output-readonly](http://codelyzer.com/rules/prefer-output-readonly)

Reply:
This is an opinion on the use of readonly which I have not seen nor want to embark on a general rule. This is probably better handled in the TypeScript configuration.

**Proposal**
Leave this rule disabled.

Reply:
An alternate take is that we should try to be as strict as possible to express our intention more clearly, even if we doubt someone would ever bother to try to assign to an output.

**Proposal**: leave this rule enabled (assuming it has an autofixer; if it doesn't, I agree it would be annoying to fix manually)

Reply:
@LukasKall this thread has a new proposal for you to consider. Let us know if you change your vote.

Reply:
If we end up 50-50, @TrevorKarjanis gets to be the deciding vote.